
From: Christoph Hellwig <hch@lst.de>

Only used by kernel/sysctl.c which absolutely can't be modular

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/proc/root.c |    3 ---
 1 files changed, 3 deletions(-)

diff -puN fs/proc/root.c~unexport-proc_sys_root fs/proc/root.c
--- 25/fs/proc/root.c~unexport-proc_sys_root	2004-09-30 22:36:50.786150936 -0700
+++ 25-akpm/fs/proc/root.c	2004-09-30 22:36:50.789150480 -0700
@@ -149,9 +149,6 @@ struct proc_dir_entry proc_root = {
 	.parent		= &proc_root,
 };
 
-#ifdef CONFIG_SYSCTL
-EXPORT_SYMBOL(proc_sys_root);
-#endif
 EXPORT_SYMBOL(proc_symlink);
 EXPORT_SYMBOL(proc_mkdir);
 EXPORT_SYMBOL(create_proc_entry);
_
